src/errors/DigitalAssetNotFound.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+
+class DigitalAssetNotFoundError extends Error {
+  address: AssetAddress
+
+  constructor(address: AssetAddress) {
+    const message = `Digital asset with address ${address} not found`
```

Comment:
this message is useless and could be easy removed
Also such `const`s better declare in file scope, not in `constructor` or any other method